docs/getting-started/quick-start.md

```diff
@@ -194,6 +200,24 @@ GROUP BY host;
 2 rows in set (0.11 sec)
 ```
 
+### Search logs by keyword
+
+Filter the log messages by keyword `timeout`:
+```sql
+SELECT * FROM app_logs WHERE lower(log_msg) @@ 'timeout' AND ts > '2024-07-11 20:00:00';
+```
+
+```sql
++---------------------+-------+------------------+-----------+--------------------+
+| ts                  | host  | api_path         | log_level | log_msg            |
++---------------------+-------+------------------+-----------+--------------------+
+| 2024-07-11 20:00:10 | host1 | /api/v1/billings | ERROR     | Connection timeout |
+| 2024-07-11 20:00:10 | host1 | /api/v1/resource | ERROR     | Connection timeout |
+| 2024-07-11 20:00:14 | host1 | /api/v1/billings | ERROR     | Timeout            |
+| 2024-07-11 20:00:14 | host1 | /api/v1/resource | ERROR     | Timeout            |
++---------------------+-------+------------------+-----------+--------------------+
+```
+
```

Comment:
It is recommended to add a link to `/user-guide/logs/query-logs.md#simple-term-matching`